sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -1761,13 +1736,13 @@ public enum SubscriptionState
     public partial class Tag
     {
         public Tag() { }
-        public System.Collections.Generic.IDictionary<string, string> TagsValue { get { throw null; } }
```

Comment:
Why the name of this property changes?

Reply:
The plurality was off, these are value(s) for a given tag